pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -472,7 +472,9 @@ public String rebalance(
       @ApiParam(value = "offline|realtime") @Nonnull @QueryParam("type") String tableType,
       @ApiParam(value = "true|false") @Nonnull @DefaultValue("true") @QueryParam("dryrun") Boolean dryRun,
       @ApiParam(value = "true|false") @DefaultValue("false") @QueryParam("includeConsuming") Boolean includeConsuming,
-      @ApiParam(value = "true|false") @DefaultValue("false") @QueryParam("downtime") Boolean downtime) {
+      @ApiParam(value = "true|false") @DefaultValue("false") @QueryParam("downtime") Boolean downtime,
+      @ApiParam(value = "number of serving replicas to keep alive per segment while rebalancing in no downtime mode")
+      @DefaultValue("1") @QueryParam("minReplicasToKeepUpForNoDowntime") Integer minReplicasToKeepUp) {
```

Comment:
Suggest "minAvailableReplicas". @snleee can you think of other possibly better param names?

Reply:
Sounds better than the current name :)

Reply:
Done.